lms/routes.py

```diff
@@ -29,6 +29,15 @@ def includeme(config):  # pylint:disable=too-many-statements
 
     config.add_route("api.assignments.create", "/api/assignment", request_method="POST")
 
+    # The gateway end-point is a bit of an oddball as it uses LTI launch as the
+    # authentication method, and so requires the LTILaunch resource
+    config.add_route(
+        "api.gateway.h.lti",
+        "/api/gateway/h/lti",
```

Comment:
I wanted to leave the possibility open of:

 * Providing access to other products
 * Providing access to `h` in ways other than `lti` launch